README.md

```
@@ -0,0 +1,17 @@
+def calculate_area_of_circle(radius):
+    """
+    Calculates the area of a circle.
+    This function contains a deliberate error for CodeRabbit AI to detect.
+    """
+    pi = 3.14159
+    # Intentional error: Incorrect formula for area (should be pi * radius**2)
```

markdownlint-cli2 (0.17.2)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)